docs/platform/custom-cloud-credentials/adding-custom-cloud-credentials.md

```diff
@@ -1,11 +1,16 @@
+import Note from "@site/src/components/Note";
+
+This documentation assumes you have followed our [prereq installation steps](./prereq-installation.md).
+
+
 ---
 description: >-
   Grid allows you to create a cluster inside your own infrastructure. Using this
   approach allows you to control where your data sits and keeping that data
   secure.
 ---
```

Comment:
<img width="1051" alt="Screen Shot 2022-03-01 at 8 57 17 AM" src="https://user-images.githubusercontent.com/13732925/156182272-5d53dd50-e769-4ccb-b1c8-a7104cc35bf5.png">

@oojo12 follow this format so that the metadata doesn't show up in the actual text. 
- description at the top (get rid of angle bracket and dash
- import Note following description
- then begin the content of the file

Reply:
addressed.